src/lib/plot/axis/axis.js

```diff
@@ -71,6 +72,10 @@ const defaultProps = {
   orientation: BOTTOM
 };
 
+const predefinedClassName = 'rv-xy-plot__axis';
+const VERTICAL_CLASS_NAME = 'rv-xy-plot__axis-vertical';
```

Comment:
i doubt it, i've also subverted that potential issue by allowing users to supply their own